public/js/test/mochitest/examples/README.md

```diff
@@ -0,0 +1,5 @@
+### Test Examples
+
+##### Pages
+* **doc_script-switching-01** - includes two scripts that reference each other. The second function has a debugger.
+* **doc-scripts** - includes three sources, a long source and two sources that reference each other.
```

Comment:
Any chance we could document them at the top of the HTML files instead? I think it'll be easier to keep up-to-date. Usually we add more edge cases to existing test files over time so they'll probably cover more than these simple descriptions. We should just r- PRs that add new examples unless it's really necessary.

If this really clarifies things for you though, I'm willing to try it.